src/index.js

```diff
@@ -79,9 +83,42 @@ export default class MarkersPlugin extends UICorePlugin {
       internalMarker.emitter.off("tooltipChanged", internalMarker.tooltipChangedHandler)
     }
     internalMarker.onDestroy()
+    this._markers.splice(index, 1)
     return true
   }
 
+  /*
+   * Clear all existing markers
+   */
+  clearMarkers() {
+    let self = this
+    if (!this._markers) {
+      return
+    } 
+    // Create a copy of the array since you cannot iterate and remove all its members correctly at the same time
+    let markersCopy = this._markers.slice(0)
```

Comment:
If we're going for efficiency you could do:
```javascript
for (let i=this._markers.length-1; i>=0; i--) {
  this._removeInternalMarker(i)
}
```
so that the array doesn't need to be cloned :)